src/cmake-tools.ts

```diff
@@ -167,6 +168,12 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
    */
   private _cmakeDriver: Promise<CMakeDriver|null> = Promise.resolve(null);
 
+  /**
+   * webview object
+   * handles everything regarding webview
```

Comment:
This comment could be better.  This object manages the CMake GUI.  I recommend referencing that explicitly.